dev/profile_startup.dart

```diff
@@ -83,6 +83,11 @@ main(List<String> args) async {
   String traces_dir = '/tmp';
 
   List<double> times = [];
+  print("Profiling startup using flutter start --trace-startup.");
+  print("Measuring from first trace event to completion of first frame upload.");
+  print("aka NativeViewGLSurfaceEGL:RealSwapBuffers.\n");
```

Comment:
pretty sure you don't need the `\n`s, they're implied by "print". Unless you want the blank lines? In which case disregard me.